website/src/components/HomepageFeatures/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import clsx from 'clsx';
+import styles from './styles.module.css';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+type FeatureItem = {
+  title: string;
+  Svg: React.ComponentType<React.ComponentProps<'svg'>>;
+  description: JSX.Element;
+};
+
+const FeatureList: FeatureItem[] = [
+  {
+    title: 'Common structures',
+    Svg: require('@site/static/img/undraw_task_re_wi3v.svg').default,
```

Comment:
I think mixing `import` and `require` is not a good idea for transpiler and bundler...

You can create a `<Svg />` component and then reuse it in other parts, what do you think?

Reply:
This is the default code of docusaurus, can you share a link to this problem just to study it, thanks.
Done, moved to a direct link.

Reply:
@mateonunez I guess we could leave it as it is as of right now, what do you think?